### PR TITLE
fix(build): moving images to bionic 18.04

### PR DESCRIPTION
**Pull request by mynktl**
Signed-off-by: mayank <mayank.patel@mayadata.io>

### CODE REVIEW

**Comment by mynktl** [comment]
**trivy openebs/cstor-istgt:ci**
```
2020-03-12T12:01:30.746+0530	INFO	Detecting Ubuntu vulnerabilities...

openebs/cstor-istgt:ci (ubuntu 18.04)
=====================================
Total: 124 (UNKNOWN: 0, LOW: 22, MEDIUM: 79, HIGH: 23, CRITICAL: 0)


```

**Comment by pawanpraka1** [comment]
@mynktl can you push an image from this branch to your docker hub (from travis). You need to setup DNAME and DPASS for your forked repo in travis and test that image and see that everything is working fine?

File: docker/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ FROM openebs/cstor-ubuntu:bionic-20200219
 RUN apt-get update; exit 0
 RUN apt-get -y install rsyslog
 RUN apt-get -y install curl tcpdump dnsutils net-tools iputils-ping gdb
-RUN apt-get -y install apt-utils libssl-dev libjson-c-dev libjemalloc-dev
+RUN apt-get -y install apt-utils libssl-dev libjson-c-dev libjemalloc-dev libssl1.0.0
```
Comment (pawanpraka1) [inline docker/Dockerfile:9]: I think it shoud be libssl-dev?

Comment (pawanpraka1) [reply docker/Dockerfile:9]: I see libssl-dev is already there, why we need  libssl1.0.0?

**Comment by mynktl** [comment]
Closing this PR since https://github.com/openebs/istgt/pull/317, which fix the same issue, has been merged.

**Pull request closed without merging**